.travis.yml

```diff
@@ -26,16 +26,15 @@ stage: test
 env:
   global:
   - COVERAGE_OPTION='./node_modules/.bin/nyc'
-  - NODE_VERSION=8.11.2
   matrix:
   - MONGODB_VERSION=3.2.13
   - MONGODB_VERSION=3.4.4
   - PARSE_SERVER_TEST_DB=postgres
   - PARSE_SERVER_TEST_CACHE=redis
-  - NODE_VERSION=stable
+  - NODE_VERSION=11.4.0
```

Comment:
why 11.4?

Reply:
It’s the next. For now stable is 10.x as they haven’t started working on 12